framework/src/main/java/com/jfcbuilder/builders/XYPolygonBuilder.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * jfreechart-builder: a builder pattern module for working with the jfreechart library
+ * 
+ * (C) Copyright 2022, by Matt E. and project contributors
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * 
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package com.jfcbuilder.builders;
+
+import java.awt.BasicStroke;
+import java.awt.Color;
+import java.awt.Paint;
+import java.awt.Stroke;
+import java.util.Arrays;
+
+import org.jfree.chart.annotations.XYAnnotation;
+import org.jfree.chart.annotations.XYPolygonAnnotation;
+
+/**
+ * Builder for producing {@link XYPolygonAnnotation} objects.
+ */
+public class XYPolygonBuilder implements IXYAnnotationBuilder<XYPolygonBuilder> {
+
+  // See XYPolygonAnnotation for source of these defaults (they're hard-coded, no constants).
+  public static final Color DEFAULT_OUTLINE_COLOR = Color.BLACK;
+  public static final Stroke DEFAULT_OUTLINE_STYLE = new BasicStroke(1.0f);
+
+  private double[] polygon;
+  private Stroke outlineStyle;
+  private Paint outlineColor;
+  private Paint fillColor;
+
+  /**
+   * Hidden constructor.
+   */
+  private XYPolygonBuilder() {
+    polygon = null;
+    outlineStyle = null;
```

Comment:
Initialize the outline members to the defaults. The polygon must be initialized by clients and the fill color can be null.